rest_framework/utils/model_meta.py

```diff
@@ -124,10 +135,17 @@ def _get_reverse_relationships(opts):
     reverse_relations = OrderedDict()
     for relation in opts.get_all_related_objects():
         accessor_name = relation.get_accessor_name()
+        # For < django 1.6
```

Comment:
I'm confused by the `For < django 1.6` comment - is the meaning there that `to_field` is only supported in 1.6+?

We always wrap any version branching behavior in `compat.py` so you probably want to create a helper function in there named `get_to_field(field)` or similar, and put this logic there.